getting-started/nuget-sso.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Telerik NuGet with SSO Account
```

Comment:
Could you consider using a title that matches the style of the other titles in the Getting Started section? For example, **Telerik NuGet and SSO Accounts**.

![image](https://user-images.githubusercontent.com/60105689/214592401-577634a2-9fdd-4e71-aa94-5f52faf0c465.png)

Alternatively, you can update the existing titles so that they use the -ing forms:
- Downloading Product Files
- Using Telerik NuGet
- Using Telerik NuGet with SSO Account